pro/src/components/Callout/Callout.tsx

```diff
@@ -41,16 +41,16 @@ const Callout = ({
   /* istanbul ignore next: graphic variations */
   switch (variant) {
     case CalloutVariant.WARNING:
-      calloutIcon = { src: strokeWarningIcon, alt: 'Attention' }
+      calloutIcon = { src: fullWarningIcon, alt: 'Attention' }
       break
     case CalloutVariant.SUCCESS:
-      calloutIcon = { src: strokeValidIcon, alt: 'Confirmation' }
+      calloutIcon = { src: fullValidIcon, alt: 'Confirmation' }
```

Comment:
Ça ne semble toujours pas être le bon picto ici.
```suggestion
      calloutIcon = { src: fullInfoIcon, alt: 'Confirmation' }
```

Reply:
il me semble que fullValidationIcon - c'est la même chose que picto "success" sur le ticket. Et, comme avant il y avait strokeValideIcon  j'ai donné cet nom fullValidateIcon. Ou il y a un autre raison? 

<img width="925" alt="Capture d’écran 2024-02-13 à 10 35 12" src="https://github.com/pass-culture/pass-culture-main/assets/115089249/58c9b158-d069-4efd-b035-6be000208a08">

Reply:
Non, désolé, je me suis trompé. Tout est bon.